dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_treemap_parents.tsx

```diff
@@ -59,41 +40,38 @@ export const ConfigTreemapParentFields = ({ dropdownList, selectedAxis, onSelect
   return (
     <>
       {!isEmpty(selectedAxis) &&
-        selectedAxis.map((_, index: number) => {
+        selectedAxis.map((obj: ParentUnitType, index: number) => {
           return (
-            <Fragment key={index}>
+            <>
               <EuiSpacer size="s" />
-              <EuiFormRow
-                label={`Parent ${index + 1}`}
-                labelAppend={
-                  <EuiText size="xs">
-                    <EuiIcon
-                      type="cross"
-                      color="danger"
-                      onClick={() => handleParentDelete(index)}
-                    />
-                  </EuiText>
-                }
-              >
-                <EuiComboBox
-                  id={uniqueId('axis-select-')}
-                  placeholder="Select a field"
-                  options={options}
-                  selectedOptions={selectedAxis[index].label !== '' ? [selectedAxis[index]] : []}
-                  isClearable={true}
-                  singleSelection={{ asPlainText: true }}
-                  onChange={(options) => handleParentChange(options, index)}
-                  aria-label="Use aria labels when no actual label is in use"
+              <EuiPanel key={index} paddingSize="s" className="panelItem_button">
```

Comment:
key is not required here. Key is required for the base element inside the loop, not the inner elements. If you are using the key for something else, then you can keep it here.